class.jetpack.php

```diff
@@ -4713,6 +4718,70 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 		return add_query_arg( $args, $api_url );
 	}
 
+	/**
+	 * This action fires at the beginning of the Manager::authorize method.
+	 */
+	public static function authorize_starting() {
+		$jetpack_unique_connection = Jetpack_Options::get_option( 'unique_connection' );
+		// Checking if site has been active/connected previously before recording unique connection.
+		if ( ! $jetpack_unique_connection ) {
+			// jetpack_unique_connection option has never been set.
+			$jetpack_unique_connection = array(
+				'connected'    => 0,
+				'disconnected' => 0,
+				'version'      => '3.6.1',
```

Comment:
Out of scope of this PR, and for another time, I wonder if this hard-coded version needs to be hard-coded.